packages/lodestar/src/metrics/metrics/beacon.ts

```diff
@@ -167,5 +167,19 @@ export function createBeaconMetrics(register: RegistryMetricCreator) {
       name: "beacon_reqresp_dial_errors_total",
       help: "Count total dial errors",
     }),
+
+    blockProductionTime: register.histogram({
+      name: "beacon_block_production_seconds",
+      help: "Full runtime of block production",
+      buckets: [0.1, 1, 10],
+    }),
+    blockProductionRequests: register.gauge({
```

Comment:
I guess because our custom type doesn't support it but there is counter metric type that should be used for values that are only incrementing. No idea whats the impact of using gauge vs counter.